client/jetpack-onboarding/main.jsx

```diff
@@ -56,6 +56,14 @@ class JetpackOnboardingMain extends React.PureComponent {
 		}
 	}
 
+	getNavigationLinkClickHandler = direction => () => {
+		const { recordJpoEvent, stepName } = this.props;
+
+		recordJpoEvent( `calypso_jpo_navigation_${ direction }_clicked`, {
```

Comment:
It's advised to programmatically generate event names, see PCYsg-4S2 (FG).

Reply:
Alternatives I can think of:

* Distinct handlers for back and forward
* One `calypso_jpo_navigation_clicked` event, with a `direction` arg

Oh BTW, `_clicked`, or `_click`? Maybe there's a convention here, too.
*

Reply:
We already handle those e.g. in
https://github.com/Automattic/wp-calypso/blob/master/client/jetpack-onboarding/steps/homepage.jsx#L23

Reply:
Good catch 👍  I've gone with the single event and a `direction` arg approach - https://github.com/Automattic/wp-calypso/pull/22377/commits/87e11249e3528903cb31f56fca22608c084bdbff